sentences/fr/homeassistant_HassSetPosition.yaml

```diff
@@ -0,0 +1,11 @@
+language: fr
+intents:
+  HassSetPosition:
```

Comment:
More a note to self: We're missing the context areas awareness, the area and the device class specific sentences but I can do that later next week it's non blocking for this pr